### DISCUSSION TITLE
Related to explore and taskbar problems on Win OS Build： 22000.2124 ARM64 Level 5.

### DISCUSSION THREAD

**Discussion by 33ddy** [WhatTheBlock/WindowsSimplify#84] [General]
StartAllBack can fix this easily. 

https://k00.fr/9vnim8py

password: 229844

**Comment by WhatTheBlock** [comment]
Thanks, but I still have to let others know about the flaws in the system itself. And I need to avoid adding third-party programs as much as possible, because some people want the system to be clean.

**Reply by biennvops** [reply]
StartAllBack is actually not a bad thing to add. However, because it is a paid software, you shouldn't add it to the prebuilt system. It is better to notice other people though
